src/network-schemas.js

```diff
@@ -161,6 +161,22 @@ function registerNetworkSchemas() {
     ]
   });
 
+  NAF.schemas.add({
+    template: "#interactable-media-frame",
```

Comment:
Later may need to add to https://github.com/mozilla/reticulum/blob/master/lib/ret_web/channels/hub_channel.ex#L695 to allow users to spawn them. (I don't think this is necessary for objects in the scene tho).